libxft.rb

```diff
@@ -15,13 +15,13 @@ class Libxft < Formula
   option "with-brewed-bzip2", "Use brewed bzip2"
   option "with-brewed-zlib", "Use brewed zlib"
 
-  depends_on "pkg-config" =>  :build
-  depends_on "fontconfig" =>  :build
+  depends_on "pkg-config" => :build
+  depends_on "fontconfig" => :build
```

Comment:
Actually, now that I look at the patchelf output from earlier, it looks like `fontconfig` might be a regular instead of `:build` dependency too.